specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-08-01/vaults.json

```diff
@@ -0,0 +1,2239 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-08-01",
```

Comment:
>"2023-08-01", [](http://example.com/codeflow?start=15&length=13)

Hi - this was a Hub generated PR ? 
Asking because the API version was chnaged only in 3rd iteration. We expect it to get chnaged in 2nd one onwards.

Reply:
Yes Vivek, it was created using Hub only.